### PR TITLE
Fix Several Bugs in Setup.sh

### PR DESCRIPTION
**Pull request by pete-tempo**

### CODE REVIEW

File: Setup.sh [hunk 1]
```diff
@@ -4,10 +4,13 @@ set -e
 
 TEMPO_ROOT=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
+cd $TEMPO_ROOT
```
Comment (pete-tempo) [inline Setup.sh:7]: This guarantees that the `git rev-parse --git-common-dir` command below is always run from Tempo, not from wherever the user ran the script from.

File: Setup.sh [hunk 2]
```diff
@@ -4,10 +4,13 @@ set -e
...
+
 if [ -z "$GIT_DIR" ]; then
 	GIT_DIR=$(git rev-parse --git-common-dir);
 	if [ $? -ne 0 ]; then
-		GIT_DIR=.git
+                echo "Failed to find .git folder"
```
Comment (pete-tempo) [inline Setup.sh:12]: We shouldn't hide this error

File: Setup.sh [hunk 3]
```diff
@@ -18,6 +21,7 @@ ADD_COMMAND_TO_HOOK() {
 
   if [ ! -f "$HOOK_FILE" ]; then
     touch "$HOOK_FILE"
+    echo -e "#!/usr/bin/env bash\n" > "$HOOK_FILE"
```
Comment (pete-tempo) [inline Setup.sh:24]: Add a shebang to the generated hook file

File: Setup.sh [hunk 4]
```diff
@@ -18,6 +21,9 @@ ADD_COMMAND_TO_HOOK() {
...
+    # https://stackoverflow.com/questions/3417896/how-do-i-prompt-the-user-from-within-a-commit-msg-hook
+    echo "exec < /dev/tty" >> "$HOOK_FILE"
```
Comment (pete-tempo) [inline Setup.sh:26]: Allow user input from the rest of the hook script. Without this, `read` commands could not get user input.

**Review by gio-tempo — APPROVED** [review]

**Pull request merged**